source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3144,6 +3181,20 @@ export class ColumnSet extends CardElement {
         }
     }
 
+    protected truncateOverflow(maxHeight: number): boolean {
+        for (let column of this._columns) {
+            column['handleOverflow'](maxHeight);
+        }
+
+        return true;
```

Comment:
Is that right? As is, ColumnSet always reports it truncated overflow, which will not be true in many cases. Shouldn't it be true only if at least one column does truncate overflow?

Reply:
As it is now, since Columns are Containers, which can always get truncated, this will always return true, even if we look at the truncation status for each column.

Reply:
Same question a for Container.